src/pages/index.js

```diff
@@ -4,11 +4,20 @@ import UserInfo from "../components/UserInfo.js";
 import Section from "../components/Section.js";
 import Card from "../components/Card.js";
 import FormValidator from "../components/FormValidator.js";
+import PopupWithConfirmation from "../components/PopupWithConfirmation.js";
 import "./index.css";
 import { initialCards, validationSettings } from "../utils/constants.js";
+import Api from "../components/Api.js";
 
+const deleteCardForm = document.querySelector("#delete-card-form");
```

Comment:
**COULD BE IMPROVED**

We can get any form using their **unique** attribute `name` or `id` from `document.forms`

```js

const profileForm = document.forms["profile-form"];
const cardForm = document.forms["card-form"];

```

You need to add `name` or `id` first in `index.html` to achieve that.

More info here https://developer.mozilla.org/en-US/docs/Web/API/Document/forms